packages/validation-manager/src/ValidationManager.ts

```diff
@@ -237,7 +237,18 @@ export class ValidationManager implements IValidationManager {
       addresses: [],
       hash: ''
     }
-    const stateOverrideForEip7702 = await this.getAuthorizationsStateOverride(userOp.authorizationList ?? [])
+    const authorizationList = getAuthorizationList(userOp)
+    if (authorizationList.length > 0) {
+      // relevant only for RIP-7562...
+      requireCond(authorizationList.length === 1, 'Only one authorization is supported', ValidationErrors.InvalidFields)
+
+      const currentChainId = BigNumber.from((this.entryPoint.provider as any)._network.chainId)
```

Comment:
Maybe just accept the current ChainID as a config parameter, there is no need to dig it up again here.